release/ray_release/environments/gce.env

```diff
@@ -1,7 +1,7 @@
 ANYSCALE_HOST=https://console.anyscale-staging.com
 ANYSCALE_CLOUD_STORAGE_PROVIDER=gs
 RELEASE_AWS_ANYSCALE_SECRET_ARN="arn:aws:secretsmanager:us-west-2:029272617770:secret:release-automation/anyscale-staging-token20221014164754935800000001-pfQunc"
-RELEASE_DEFAULT_CLOUD_ID="cld_tPsS3nQz8p5cautbyWgEdr4y"
+RELEASE_DEFAULT_CLOUD_ID="cld_vy7xqacrvddvbuy95auinvuqmt"
 RELEASE_DEFAULT_PROJECT="prj_qC3ZfndQWYYjx2cz8KWGNUL4"
 ANYSCALE_PROJECT="prj_qC3ZfndQWYYjx2cz8KWGNUL4"
 GOOGLE_CLOUD_PROJECT="anyscale-oss-ci"
```

Comment:
Is this GCP project still accurate?

Reply:
It's the name of the GCP account; I'm not sure if the new cloud is using a new account, maybe not. We'll see if tests pass

Reply:
The old and new cloud use `anyscale-bridge-cd812d38`

Reply:
But it looks like permissions are configured for some Storage buckets in `anyscale-oss-ci`, so it's probably fine :)

Reply:
Sound good! I think that'll work, the anyscale-bridge-cd812d38 seems to be the admin, all powerful account